salt/hg/config/repos.conf

```diff
@@ -15,5 +15,5 @@ logourl = https://hg.python.org
 pygments_style = default
 style = hgpythonorg
 
-[extensions]
-hgext.highlight =
```

Comment:
Need to understand the impact of this - it prevented service loading for `hg` because of `pygments` package issue.. that may not be a big deal since this is a most old reference?

Reply:
I wonder if adding python3-pygments to the https://github.com/python/psf-salt/blob/6fcd4e79fcec9a1e5497e5653e3874e1edad6b6c/salt/hg/init.sls#L1 would allow it to be restored.